README.md

```diff
@@ -2,6 +2,9 @@
 
 This application allows residents of Amsterdam to easily see if they need a permit for a construction activity for their specific building. For various activities, like installing a window or solar panels on the roof or extending the house, we ask a number of questions about the location and plans. This leads to an outcome, either license-free or licensing obligation, where it is clear which questions lead to this conclusion. This tool can therefore be used to see how in a specific location you could carry out a construction activity without the requirement of a permit.
 
+[![codecov](https://codecov.io/gh/Amsterdam/vergunningcheck/branch/develop/graph/badge.svg)](https://codecov.io/gh/Amsterdam/vergunningcheck)
+![Github CI](https://github.com/Amsterdam/vergunningcheck/workflows/Github%20CI/badge.svg)
+
```

Comment:
Can you put this in the bottom in the `## Thanks to` section? Or do they really require it to be in the top?

Reply:
Positioning was odd, I agree.
Looking at other repo's in the community (like React and CRA) they add their badges to the h1 / title of the markdown. I moved it there.